pyuvdata/utils.py

```diff
@@ -3975,6 +4013,195 @@ def check_lsts_against_times(
         )
 
 
+def uvw_track_generator(
+    lon_coord=None,
+    lat_coord=None,
+    coord_frame="icrs",
+    coord_epoch=None,
+    coord_type="sidereal",
+    time_array=None,
+    telescope_loc=None,
+    antenna_frame="itrs",
+    antenna_positions=None,
+    antenna_numbers=None,
+    ant_1_array=None,
+    ant_2_array=None,
+    uvw_array=None,
+    force_postive_u=False,
+):
+    """
+    Calculate uvw coordinates (among other values) for a given position on the sky.
+
+    This function is meant to be a user-friendly wrapper around several pieces of code
+    for effectively simulating a track. Note that the naming of things may change
+    before this hits a PR.
```

Comment:
This should probably be updated before we merge 😉

Reply:
Heh, whoops -- I've taken out that bit of the docstring.